src/components/createInstance.js

```diff
@@ -82,7 +121,8 @@ class CreateInstance extends Component {
       rootFolderName,
       logo,
       rolesAllowed,
-      maxSpace
+      maxSpace,
+      extraSpaceOptions
     } = this.state.formObj
     const { createFilemanager } = this.props
     if (filemanagerName && filemanagerUrlPath && logo && rolesAllowed) {
```

Comment:
add conditions of extra space here as it is a required field and form should submit with empty storage options or create default options